cms/settings/base.py

```diff
@@ -1003,6 +1005,13 @@
     "Page",
 }
 
+# Allow-list of language codes whose pages are included in search indexing
+SEARCH_INDEX_INCLUDED_LANGUAGES = [
+    code.strip().lower()
+    for code in env.get("SEARCH_INDEX_INCLUDED_LANGUAGES", LANGUAGE_CODE).split(",")
+    if code.strip() and code.strip().lower() in _valid_language_codes
+]
```

Comment:
I think instead of silently ignoring invalid language codes, this should be a system check. If we incorrectly configured it, the search index would essentially be disabled, and we wouldn't know until someone spotted it.